test/suite/test_rollback_to_stable14.py

```diff
@@ -81,6 +81,10 @@ def test_rollback_to_stable(self):
         value_modR = mod_val(value_modQ, 'R', 1)
         value_modS = mod_val(value_modR, 'S', 2)
         value_modT = mod_val(value_modS, 'T', 3)
+        value_modW = mod_val(value_modT, 'W', 4)
+        value_modX = mod_val(value_modW, 'X', 5)
+        value_modY = mod_val(value_modX, 'Y', 6)
+        value_modZ = mod_val(value_modY, 'Z', 7)
```

Comment:
It looks like `value_modW` and `value_modY` are not used in this test. Is that the intention?